lib/client/api_login_test.go

```diff
@@ -144,13 +146,15 @@ func TestTeleportClient_Login_local(t *testing.T) {
 
 	ctx := context.Background()
 	tests := []struct {
-		name             string
-		secondFactor     constants.SecondFactorType
-		inputReader      *prompt.FakeReader
-		solveWebauthn    func(ctx context.Context, origin string, assertion *wanlib.CredentialAssertion, prompt wancli.LoginPrompt) (*proto.MFAAuthenticateResponse, error)
-		authConnector    string
-		allowStdinHijack bool
-		preferOTP        bool
+		name                    string
+		secondFactor            constants.SecondFactorType
+		inputReader             *prompt.FakeReader
+		solveWebauthn           func(ctx context.Context, origin string, assertion *wanlib.CredentialAssertion, prompt wancli.LoginPrompt) (*proto.MFAAuthenticateResponse, error)
+		authConnector           string
+		allowStdinHijack        bool
+		preferOTP               bool
+		hasCredentials          bool
```

Comment:
Same if you take the other comment:

```suggestion
		hasTouchIDCredentials          bool
```